strr-host-pm-web/app/pages/application.vue

```diff
@@ -1,7 +1,6 @@
 <script setup lang="ts">
 import { ConnectStepper, FormReview } from '#components'
-const { t } = useI18n()
-const route = useRoute()
+const { t } = useNuxtApp().$i18n
```

Comment:
Any reason to do it this way?

Reply:
This is so the content is pulled from translation file for the tests.